general/base_import/import_faq.rst

```diff
@@ -0,0 +1,201 @@
+----------------------------------------------------------
+
+Some fields define a relationship with another object. For example, the country of a contact is a link to a record of the 'Country' object. When you want to import such fields, Odoo will have to recreate links between the different records. To help you import such fields, Odoo provides 3 mechanisms. You must use one and only one mechanism per field you want to import.
+
+For example, to reference the country of a contact, Odoo proposes you 3 different fields to import: 
+
+- Country: the name or code of the country
+- Country/Database ID: the unique Odoo ID for a record, defined by the ID postgresql column
+- Country/External ID: the ID of this record referenced in another application (or the .XML file that imported it) 
+
+For the country Belgium, you can use one of these 3 ways to import: 
+
+- Country: Belgium
+- Country/Database ID: 21
+- Country/External ID: base.be
+
+According to your need, you should use one of these 3 ways to reference records in relations. Here is when you should use one or the other, according to your need: 
+
+- Use Country: This is the easiest way when your data come from CSV files that have been created manually.
+- Use Country/Database ID: You should rarely use this notation. It's mostly used by developers as it's main advantage is to never have conflicts (you may have several records with the same name, but they always have a unique Database ID)
+- Use Country/External ID: Use External ID when you import data from a third party application.
+
+When you use External IDs, you can import CSV files with the "External ID" column to define the External ID of each record you import. Then, you will be able to make a reference to that record with columns like "Field/External ID". The following two CSV files give you an example for Products and their Categories.
+
+`CSV file for categories <../../_static/example_files/External_id_3rd_party_application_product_categories.csv>`_.
+
+`CSV file for Products <../../_static/example_files/External_id_3rd_party_application_products.csv>`_.
+
+
+What can I do if I have multiple matches for a field?
+-----------------------------------------------------
+
+If for example you have two product categories with the child name "Sellable" (ie. "Misc. Products/Sellable" & "Other Products/Sellable"), your validation is halted but you may still import your data. However, we recommend you do not import the data because they will all be linked to the first 'Sellable' category found in the Product Category list ("Misc. Products/Sellable"). We recommend you modify one of the duplicates' values or your product category hierarchy.
+
+However if you do not wish to change your configuration of product categories, we recommend you use make use of the external ID for this field 'Category'.
+
+
+
+How can I import a many2many relationship field (e.g. a customer that has multiple tags)?
```

Comment:
Do not put the "e.g." in the title, way too long (but can be put in the sentence below)